source/_cookbook/configuration_yaml_by_aneisch.markdown

```diff
@@ -8,6 +8,6 @@ comments: false
 sharing: true
```

Comment:
Please remove this file. It has nothing to do with this PR.

Create a separate PR for this.